administrator/components/com_admin/sql/updates/mysql/5.0.0-2023-08-30.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE `#__extensions`
+   SET `locked` = 0
+ WHERE `element` = 'recaptcha_invisible' AND `folder` = 'captcha';
```

Comment:
```suggestion
 WHERE `type` = 'plugin' AND `element` = 'recaptcha_invisible' AND `folder` = 'captcha';
```
We should be precise when identifying extensions.